install_config/registry/accessing_registry.adoc

```diff
@@ -305,4 +306,42 @@ openshift_registry_request_duration_seconds_count{name="test/origin-pod",operati
 <1> `<user>` can be arbitrary, but `<secret>` must match the value specified in the
 xref:../registry/extended_registry_configuration.adoc#docker-registry-configuration-reference-openshift[registry configuration].
 
+Another way how to access metrics is to use a cluster role. You still need to enable endpoint,
```

Comment:
s/how//
s/access metrics/access the metrics/
s/enable endpoint/enable the endpoint/

Reply:
@bparees done